dokka-subprojects/plugin-versioning/src/main/kotlin/org/jetbrains/dokka/versioning/VersioningConfiguration.kt

```diff
@@ -23,7 +23,9 @@ public data class VersioningConfiguration(
         if (olderVersionsDir != null)
             assert(olderVersionsDir!!.isDirectory) { "Supplied previous version $olderVersionsDir is not a directory!" }
 
-        return olderVersionsDir?.listFiles()?.toList().orEmpty() + olderVersions.orEmpty()
+        return olderVersionsDir?.listFiles()
+            ?.filter { it.isDirectory && !it.isHidden }
```

Comment:
Do we really need to filter out `hidden` directories here? Their definition is a bit vague, and I think that getting just directories should be enough.

If you think, that we should still filter out `hidden` directories for some reason, then it would be nice to add a comment for reasoning there.

Reply:
My reasoning was that typically a hidden directory wouldn't be used to store a versioned docs archive, but could be some system created or metadata directory. While .DS_Store was the cause of the issue I ran into and would be filtered out by just ignoring non-directory files, I was thinking there could be other hidden system or metadata files that are directories. An example might be .git or less likely .idea.

If you think it's more likely valid version directories actually are hidden, we could remove the `!it.isHidden` check. I'm fine either way.